inno_compile-script

```diff
@@ -0,0 +1,33 @@
+#define MyAppName "Azure DevOps Plugin for ManicTime"
+#define MyAppVersion "0.1"
+#define MyAppPublisher "mikefitz888"
+#define MyAppURL "https://github.com/mikefitz888/Azure-Manic-7Pace"
+
+[Setup]
+; NOTE: The value of AppId uniquely identifies this application.
+; Do not use the same AppId value in installers for other applications.
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+AppId={{447729A4-3AA8-4B49-B6DB-047D48668268}
+AppName={#MyAppName}
+AppVersion={#MyAppVersion}
+;AppVerName={#MyAppName} {#MyAppVersion}
+AppPublisher={#MyAppPublisher}
+AppPublisherURL={#MyAppURL}
+AppSupportURL={#MyAppURL}
+AppUpdatesURL={#MyAppURL}
+DefaultDirName={localappdata}\Finkit\ManicTime\Plugins\MFitzpatrick.TagSource.Azure7PacePlugin
```

Comment:
This should be `...\Plugins\Packages\MFitz...`

Reply:
Updated to the correct path added in updated PR